mpris2controller

```diff
@@ -215,7 +236,12 @@ def is_daemon_up():
 def call_daemon_method(method_name):
     try:
         log.info('Calling method %s on daemon.', method_name)
-        getattr(dbus.SessionBus().get_object(MY_BUS_NAME, MY_PATH), method_name)()
+        result = getattr(dbus.SessionBus().get_object(MY_BUS_NAME, MY_PATH), method_name)()
+        if result:
+            try:
+                print(json.dumps(dict(result)))
```

Comment:
Might be better to check for DBUS's return type (`dbus.Dictionary`, `dbus.Array`, etc.) and display accordingly (instead of trying only for `dict`). I'm assuming `json.dumps` is rejecting the `dbus.Dictionary` directly, otherwise we should definitely just let `json.dumps` deal with it.

Reply:
> I'm assuming `json.dumps` is rejecting the `dbus.Dictionary` directly, otherwise we should definitely just let `json.dumps` deal with it.

It is not rejecting it, but then `print(json.dumps(result))` works on `str`/`dbus.String` returned by `Status` method (`ValueError` is not thrown), and prints _quoted_ PlaybackStatus.

You're probably right that it might be better to check for DBUS's return type, but isn't it against common Python practice to not use explicit type-checking (in favor of duck-typing)?

Reply:
Hmm. You're right. I'm not sure I like it either way, and we should probably redesign `call_daemon_method` to be a little less generic, but this should work for now :P